nengo_spinnaker/operators/lif.py

```diff
@@ -234,17 +265,31 @@ def after_simulation(self, netlist, simulator, n_steps):
 
                 simulator.data[p] = probed_spikes[::sample_every, neuron_slice]
 
+        # If profiling is enabled
+        if self.profiler_region.n_samples > 0:
+            # Loop through vertices
+            for vertex in self.vertices:
+                # Get profiler output memory block
+                mem = self.profiler_output_mem[vertex]
+                mem.seek(0)
+
+                # Read profiler data from memory and put somewhere accessible
+                simulator.profiler_data[self.ensemble] =\
```

Comment:
Worth noting that for `sim.run(t)` if `t > sim.period` this line will be called multiple times and will overwrite stored values.  Don't know if that's a concern provided we know how long our experiments will be.